src/webui/www/private/scripts/download.js

```diff
@@ -125,7 +125,7 @@ window.qBittorrent.Download = (function() {
                 const category = categories[categoryName];
                 let savePath = defaultSavePath;
                 if (category !== undefined)
-                    savePath = (category['savePath'] !== "") ? category['savePath'] : (defaultSavePath + categoryName);
+                    savePath = (category['savePath'] !== "") ? category['savePath'] : `${defaultSavePath}/${categoryName}`;
```

Comment:
Such a notation looks unusual to me (I'm not an active JS code).
@Chocobo1?

Reply:
It's a template string, which its use is a JS convention for strings requiring interpolation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation